docs/reference/filebeat/filebeat-input-filestream.md

```diff
@@ -560,6 +560,26 @@ The `clean_inactive` configuration option is useful to reduce the size of the re
 
 This config option is also useful to prevent Filebeat problems resulting from inode reuse on Linux. For more information, see [Inode reuse causes Filebeat to skip lines](/reference/filebeat/inode-reuse-issue.md).
 
+To disable, set `clean_inactive: -1`. {applies_to}`stack: ga 9.2.0`Setting it to zero will also
+disable it.
+
+::::{warning}
+For versions `<= 9.2.0` setting `clean_inactive: 0` will cause
+Filebeat to re-ingest all files on restart.
+::::
```

Comment:
Maybe something like this to avoid having to show information about a future version on the rendered page?

```suggestion
To disable, set `clean_inactive` to either:
* `-1`
* {applies_to}`stack: ga 9.2.0` `0`
  ::::{warning}
  In earlier versions, setting `clean_inactive: 0` will cause
  Filebeat to re-ingest all files on restart.
  ::::
```